bgpd/rfapi/rfapi_monitor.c

```diff
@@ -1273,7 +1273,7 @@ static void rfapiMonitorEthDetachImport(
 #if DEBUG_L2_EXTRA
 	char buf_prefix[PREFIX_STRLEN];
 
-	prefix2str(&rn->p, buf_prefix, sizeof(buf_prefix));
+	prefix2str(agg_node_get_prefix(rn), buf_prefix, sizeof(buf_prefix));
```

Comment:
can't this be handled the same way as `bgp_node_get_prefix`? I mean get rid of `char buf_prefix[PREFIX_STRLEN];` and just use `%pRN` below. Doesn't it work with struct agg_node?

Reply:
when I moved this stuff to use %pRN shit broke yo :( in the topotests.  decided to put back and not massage.  This had to do with the rfapi stuff using non-FRR standard output.